log/gps/gps_mgl_enigma_waypoint.ksy

```diff
@@ -0,0 +1,134 @@
+  application: Enigma software
+  file-extension:
+    - ewd # WAYPOINT.EWD usually
+    - rte # MYROUTE.RTE usually
+  license: Unlicense
+  endian: le
+  encoding: ascii
+
+-license-header: | # the conditions contradict each other and seem to be unenforceable
+  MGL Avionics places this document and the data format it describes in the public domain in order to foster the use of a common, compact waypoint and route file format that is particularly optimized for use with resource limited systems and/or systems that require fast access to individual items in the waypoint files in order to increase overall systems performance.
+  In order to allow recognition of the format any party adopting this format agrees to refer to this format as the “Enigma waypoint format”. All data using this format should be considered to be in the public domain and the implementer will make reasonable effort to allow any interested party to use the data.
+  Exceptions:
+    MGL Avionics reserves copyright and denies use of this data format for any military or related purpose.
+
+doc: |
+  Waypoint format used by Enigma series of EFIS instruments from MGL Avionics.
+  It is common to create an index file containing a simple list of indexes, one to each entry in the waypoint file, sorted by a particular criteria (for example distance of the waypoint to a given location), rather than sorting the actual waypoint list itself. This is a method commonly employed by database applications.
+
+doc-ref:
+  - https://mglavionics.co.za/Docs/Enigma%20Waypoint%20format.pdf
+
+seq:
+  - id: waypoints
+    type: waypoint
+    repeat: eos
+    doc: There is no demand that waypoints are stored in any particular order in the waypoint file. They might be sorted by short name or waypoint type but this is of little consequence. The order of waypoints in the file matters only for route files. Route waypoints are stored in the file in order of their appearance in the route. A route should have at least one waypoint.
+
+types:
+  waypoint:
+    seq:
+      - id: lat
+        type: sph_coord
+        doc: N is positive
+      - id: lon
+        type: sph_coord
+        doc: E is positive
+      - id: data_raw
+        type: u4
+      - id: type
+        type: u1
+        enum: type
+      - id: name_short
+        type: dc_str(6)
+        doc: This entry is used as “key” and in aviation terms would contain the short identifier of an airfield or navaid beacon. This key is used to lookup related data in other databases, for example an airport data base containing details of an airport such as frequencies and runway headings.
+      - id: name_long
+        type: dc_str(27)
+        doc: |
+          The long name contains a human readable description of the waypoint.
+          For example: “Los Angeles International”. The long name is arbitrary and not used as search key for any purpose."
+
+    instances:
+      is_data_frequency:
+        value: 25 >= type.to_i and type.to_i >= 9
```

Comment:
Maybe this is personal preference, but I think it's more readable this way:

```suggestion
        value: type.to_i >= 9 and type.to_i <= 25
```

Also it would probably be good to explain what exactly determines the format of the data and what it means. As I understand it, for navaids it's a radio frequency and for everything else it's an altitude? Is it correct that for the `unspecified` type and for unknown/not yet defined types the data is known to be an altitude, or could it be something different/an unknown data format?

Reply:
@dgelessus 

> Maybe this is personal preference, but I think it's more readable this way

I think exactly the same thing and also wanted to point it out. So at least, it is personal preference of both of us 🙂

Reply:
I feel like KS needs the syntax like in python for chained comparisons, so it should look like `25 >= type.to_i >= 9` or `9 <= type.to_i <= 25` and translate to the chained comparisons for python.

>As I understand it, for navaids it's a radio frequency and for everything else it's an altitude?

For `unspecified` it is altitude, and for 1 of the types it is undefined at all, according to the PDF.